vision.md

```diff
@@ -0,0 +1,115 @@
+
+
+ ## What is our vision?
+
+The aim of creating thin-edge.io is to provide a IoT edge device framework for IoT project teams which makes it easy to enable resource constrained devices for IoT. Unlike other solutions, we are not just another single-purpose agent but a flexible platform with re-usable components without any vendor lock-in, focused on adressing the needs of both IT and OT users. 
+
+## What is our motivation?
+
+While, we believe that IoT (edge/device management, middleware, data analytics) are the driving forces of the fourth industrial revolution, where companies are forced to leverage IoT solutions to stay competitive, we are observing that a large amount of companies are suffering from failed IoT projects and initiatives, where a lot of time and money is spent on device enablement (e.g. connectivity, security, device management).
```

Comment:
I think this is very defensively written. I think being bolder on the statements would make it clearer:


```suggestion
We believe that IoT (edge/device management, middleware, data analytics) are the driving forces of the fourth industrial revolution: companies are forced to leverage IoT solutions to stay competitive. In the past a large amount of companies have suffered from failed IoT projects and initiatives, where a lot of time and money was spent on device enablement: connectivity, security, device management, etc...
```